__tests__/app.test.js

```diff
@@ -174,3 +174,46 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
```

Comment:
We need another 200 status for when an article exists but doesn't have comments yet, in that case we should see an empty array (it's not an error to have 0 comments)